cmd/cli/multiregion/setup.go

```diff
@@ -0,0 +1,237 @@
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database", Value: pFlags.org + "/" + databaseWorkspace(pFlags)},
+	}
+	setVars(pFlags, databaseSecondaryWorkspace(pFlags), databaseVars)
+
+	pmaVars := []lib.TFVar{
+		{Key: "pma_subdomain", Value: pFlags.idp + "-pma-secondary"},
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database_secondary", Value: pFlags.org + "/" + databaseSecondaryWorkspace(pFlags)},
+	}
+	setVars(pFlags, pmaSecondaryWorkspace(pFlags), pmaVars)
+
+	emailVars := []lib.TFVar{
+		{Key: "email_subdomain", Value: pFlags.idp + "-email-secondary"},
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database_secondary", Value: pFlags.org + "/" + databaseSecondaryWorkspace(pFlags)},
+	}
+	setVars(pFlags, emailSecondaryWorkspace(pFlags), emailVars)
+
+	brokerVars := []lib.TFVar{
+		{Key: "broker_subdomain", Value: pFlags.idp + "-broker-secondary"},
+		{Key: "mfa_totp_apibaseurl", Value: "TODO"},     // TODO: get this value
+		{Key: "mfa_webauthn_apibaseurl", Value: "TODO"}, // TODO: get this value
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database_secondary", Value: pFlags.org + "/" + databaseSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_email_secondary", Value: pFlags.org + "/" + emailSecondaryWorkspace(pFlags)},
+	}
+	setVars(pFlags, brokerSecondaryWorkspace(pFlags), brokerVars)
+
+	pwVars := []lib.TFVar{
+		{Key: "api_subdomain", Value: pFlags.idp + "-pw-api-secondary"},
+		{Key: "ui_subdomain", Value: pFlags.idp + "-pw-secondary"},
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database_secondary", Value: pFlags.org + "/" + databaseSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_email_secondary", Value: pFlags.org + "/" + emailSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_broker_secondary", Value: pFlags.org + "/" + brokerSecondaryWorkspace(pFlags)},
+	}
+	setVars(pFlags, pwSecondaryWorkspace(pFlags), pwVars)
+
+	sspVars := []lib.TFVar{
+		{Key: "ssp_subdomain", Value: pFlags.idp + "-secondary"},
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_database_secondary", Value: pFlags.org + "/" + databaseSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_broker_secondary", Value: pFlags.org + "/" + brokerSecondaryWorkspace(pFlags)},
+		{Key: "tf_remote_pwmanager_secondary", Value: pFlags.org + "/" + pwSecondaryWorkspace(pFlags)},
+	}
+	setVars(pFlags, sspSecondaryWorkspace(pFlags), sspVars)
+
+	syncVars := []lib.TFVar{
+		{Key: "sync_subdomain", Value: pFlags.idp + "-sync-secondary"},
+		{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
```

Comment:
At some point, it might be nice to have these all created in one go above, instead of duplicating the same structs several times.

Reply:
I guess I don't see the simplification option you describe. Each struct is slightly different. Or are you talking about the inner struct? (`{Key: "tf_remote_pwmanager_secondary", Value: pFlags.org + "/" + pwSecondaryWorkspace(pFlags)}`)

Reply:
E.g.
You have this in there 8 times ...
```
{Key: "tf_remote_cluster_secondary", Value: pFlags.org + "/" + clusterSecondaryWorkspace(pFlags)},
```